test/e2e/framework/assertions.go

```diff
@@ -191,7 +191,7 @@ func (f *Framework) GetOperatorPod(t *testing.T) *v1.Pod {
 
 	// get the operator deployment
 	operator := appsv1.Deployment{}
-	f.AssertResourceEventuallyExists("monitoring-stack-operator", "operators", &operator)(t)
+	f.AssertResourceEventuallyExists("monitoring-stack-operator-prometheus-operator", "operators", &operator)(t)
```

Comment:
I am not sure why this is needed 🤔 but lets see if CI agrees :)

Reply:
This test fails already if I am not mistaken. The _only_ way I could get the tests to pass is by changing that. I just checked again on a clean repository